Flexible storage (#653)
neo4j/docs-aura: modules/ROOT/pages/managing-instances/instance-resources.adoc

== Storage

Amount of disk space reserved to store data, in gigabytes (GB). 
Ideally, the database should all fit into memory (page cache) for the best performance. 

> Keep an eye on this metric to make sure you have enough storage for future growth. 

Author: `Storage` graph
"Keep an eye on the storage graph to make sure you have enough storage for future growth. 
The graph is available in *Metrics > Resources > Storage*."

> Keep an eye on this metric to make sure you have enough storage for future growth. 

Reviewer: It's a bit confusing to refer to this as a graph.

== Storage

Amount of disk space reserved to store data, in gigabytes (GB). 
Ideally, the database should all fit into memory (page cache) for the best performance. 
Keep an eye on storage metrics to make sure you have enough storage for future growth. 

> The graph is available in *Metrics > Resources > Storage*.

Reviewer: ```suggestion
This metric is available in *Metrics > Resources > Storage*.
```
